client/tokenized-payment-request/cart-api.js

```diff
@@ -0,0 +1,131 @@
+/* global jQuery */
+
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { applyFilters } from '@wordpress/hooks';
+
+export default class PaymentRequestCartApi {
+	// Used on product pages to interact with an anonymous cart.
+	// This anonymous cart is separate from the customer's cart, which might contain additional products.
+	// This functionality is also useful to calculate product/shipping pricing (and shipping needs)
+	// for compatibility scenarios with other plugins (like WC Bookings, Product Add-Ons, WC Deposits, etc.).
+	cartRequestHeaders = {};
+
+	async placeOrder( paymentData ) {
+		return await apiFetch( {
+			method: 'POST',
+			path: '/wc/store/v1/checkout',
+			credentials: 'omit',
+			headers: {
+				'X-WC-Payments-prb-request': true,
+				...this.cartRequestHeaders,
+			},
+			data: paymentData,
+		} );
+	}
+
+	async getCart() {
```

Comment:
```suggestion
	/**
	 * Returns customer's current default cart object.
	 *
	 * @returns {Promise} Cart response object
	 */
	async getCart() {
```
This is another irritating suggestion and this isn't something that is explicitly mentioned in our JS coding standards either, but could we add some documentation comments to these functions? I think most of these functions are pretty self-explanatory, but referring to this function particularly, I had to have a bit more understanding of the code and the Store API to understand that this function cannot be used to gather the current temporary/tokenized cart, but that it will return the customer's existing, default cart object instead. 

(Plus, providing comprehensive function documentation seems to be a nice precedent to follow that is generally maintained with most of our more recent client code (sorry! >_<).)

Reply:
([Maybe it is in our coding standards actually](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#what-should-be-documented), soz.)

Reply:
That's totally fair and a great observation, since the `PaymentRequestCartApi` is meant to be an interface to the Cart object in the Store API.